climakitae/core/data_export.py

```diff
@@ -855,12 +858,13 @@ def export(
     # now here is where exporting actually begins
     # we will have different functions for each file type
     # to keep things clean-ish
-    if "zarr" == req_format:
-        _export_to_zarr(data, save_name, mode)
-    if "netcdf" == req_format:
-        _export_to_netcdf(data, save_name)
-    elif "csv" == req_format:
-        _export_to_csv(data, save_name)
+    match req_format:
+        case "zarr":
+            _export_to_zarr(data, save_name, mode)
+        case "netcdf":
+            _export_to_netcdf(data, save_name)
+        case "csv":
+            _export_to_csv(data, save_name)
```

Comment:
final case

Reply:
`req_format` is already error trapped before getting to this `match` but I will add the final case for grins.

Reply:
Fixed in [e0329e5](https://github.com/cal-adapt/climakitae/pull/564/commits/e0329e5463f65abca87b6df45a7197668ff2d23a)